Gruntfile.js

```diff
@@ -220,6 +222,14 @@ module.exports = function(grunt) {
                     }
                 }
             }
+        },
+        eslint: {
+            options: {
+                config: '.eslintrc',
+                reset: true
+            },
+            target: ['ui/js/*.js',
+                     'ui/js/**/*.js']
```

Comment:
Think we also need `ui/js/**/**/*.js` to include the deeper subdirectories under the `directives` directory.